include/openssl/aes.h

```diff
@@ -45,68 +45,67 @@ struct aes_key_st {
-                                    const int enc))
-DEPRECATEDIN_3_0(void
-                 AES_cfb1_encrypt(const unsigned char *in, unsigned char *out,
-                                  size_t length, const AES_KEY *key,
-                                  unsigned char *ivec, int *num, const int enc))
-DEPRECATEDIN_3_0(void
-                 AES_cfb8_encrypt(const unsigned char *in, unsigned char *out,
-                                  size_t length, const AES_KEY *key,
-                                  unsigned char *ivec, int *num, const int enc))
-DEPRECATEDIN_3_0(void
-                 AES_ofb128_encrypt(const unsigned char *in, unsigned char *out,
-                                    size_t length, const AES_KEY *key,
-                                    unsigned char *ivec, int *num))
+# ifndef OPENSSL_NO_DEPRECATED_3_0
+OSSL_DEPRECATEDIN_3_0 const char *AES_options(void);
+OSSL_DEPRECATEDIN_3_0
+int AES_set_encrypt_key(const unsigned char *userKey, const int bits,
+                        AES_KEY *key);
+OSSL_DEPRECATEDIN_3_0
+int AES_set_decrypt_key(const unsigned char *userKey, const int bits,
+                        AES_KEY *key);
+OSSL_DEPRECATEDIN_3_0
+void AES_encrypt(const unsigned char *in, unsigned char *out,
+                 const AES_KEY *key);
+OSSL_DEPRECATEDIN_3_0
+void AES_decrypt(const unsigned char *in, unsigned char *out,
+                 const AES_KEY *key);
+OSSL_DEPRECATEDIN_3_0
+void AES_ecb_encrypt(const unsigned char *in, unsigned char *out,
+                     const AES_KEY *key, const int enc);
+OSSL_DEPRECATEDIN_3_0
+void AES_cbc_encrypt(const unsigned char *in, unsigned char *out,
+                     size_t length, const AES_KEY *key,
+                     unsigned char *ivec, const int enc);
+OSSL_DEPRECATEDIN_3_0
+void AES_cfb128_encrypt(const unsigned char *in, unsigned char *out,
+                        size_t length, const AES_KEY *key,
+                        unsigned char *ivec, int *num, const int enc);
+OSSL_DEPRECATEDIN_3_0
+void AES_cfb1_encrypt(const unsigned char *in, unsigned char *out,
+                      size_t length, const AES_KEY *key,
+                      unsigned char *ivec, int *num, const int enc);
+OSSL_DEPRECATEDIN_3_0
+void AES_cfb8_encrypt(const unsigned char *in, unsigned char *out,
+                      size_t length, const AES_KEY *key,
+                      unsigned char *ivec, int *num, const int enc);
+OSSL_DEPRECATEDIN_3_0
+void AES_ofb128_encrypt(const unsigned char *in, unsigned char *out,
+                        size_t length, const AES_KEY *key,
+                        unsigned char *ivec, int *num);
+# endif
 
 /* NB: the IV is _two_ blocks long */
-DEPRECATEDIN_3_0(void
-                 AES_ige_encrypt(const unsigned char *in, unsigned char *out,
-                                 size_t length, const AES_KEY *key,
-                                 unsigned char *ivec, const int enc))
+# ifndef OPENSSL_NO_DEPRECATED_3_0
```

Comment:
Closing and reopening the `# ifndef` should not be necessary for just the comment?

Reply:
Thanks.  I didn't notice that one.  `util/fix-deprecation` is only so smart